src/tests/tic-tac-toe.test.ts

```diff
@@ -0,0 +1,21 @@
+import Game from "../tic-tac-toe";
+
+describe("Tic-tac-toe", () => {
+  let game: Game;
+  beforeEach(() => {
+    game = new Game();
+  });
+  it("starts with an empty board", () => {
+    const board = [["", "", ""], ["", "", ""], ["", "", ""]];
+    expect(game.board).toEqual(board);
+  });
+  it("places an X on the board on X's turn", () => {
```

Comment:
How do we know it's X's turn, here?

Reply:
Maybe this should just be testing that we're able to mark the board at all. The next test checks that the player is switched each turn, so maybe it's best to describe this assertion as making sure we can mark the board, and let the next test deal with which player's turn it is?

Reply:
Yeah, I always waffle on these kinds of "redundant" tests, but my preference is usually to have clear test descriptions (even if the test code itself isn't quite as clear).

Even just changing the description here to `it("starts on the X players turn")` and changing nothing else would be fine; my comment was mainly about clear intent.